api/scores_justifications/routes.py

```diff
@@ -0,0 +1,51 @@
+from api.responses import error_response
+from api.responses import scores_justifications_response
+from api.responses import scores_justifications_response_list
+from db.models.scores_justifications import ScoresJustificationsError
+from db.models.scores_justifications import ScoresJustificationsMethods
+from flask.views import MethodView
+
+
+class ScoresJustificationsView(ScoresJustificationsMethods, MethodView):
```

Comment:
Im just pondering directory/file structure here. I understand it mirrors our db model classes and is called routes.py because our api.yml routes are defined here, but would it make more sense for these files be inside a routes folder within api rather than api directory filled with directories based on the different database models, which will be different for each repo. If im missing the point please could we have a call :)

Reply:
that is a very good point! was thinking we should have a chat about this also and agree on the best way of doing this 👍🏾